ctx.py

```diff
@@ -203,11 +203,13 @@ def cfg_factory():
     return factory
 
 
-def configure_default_logging(stdout_level=None):
+def configure_default_logging(stdout_level=None, whd_log_file_path=None):
     import logging
     import logging.config
     if not stdout_level:
         stdout_level = logging.INFO
+    if not whd_log_file_path:
+        whd_log_file_path = os.path.join('/', 'tmp', 'whd_log')
```

Comment:
please do not do that.
if there is not logfile passed, there is not logging to a logfile